### DISCUSSION TITLE
请问如何用ventoy来重装MacOS？

### DISCUSSION THREAD

**Discussion by shuangfu** [ventoy/Ventoy#2645] [Q&A]
抱歉，本人英文不好，这里用中文描述。

**以下需求都是针对白苹果，而不是黑苹果**

需求：我希望通过Ventoy U盘给我的macbook pro 13(2019版)重装MacOS。

问题描述：通过网络恢复系统速度太慢，于是我下载了引导版的MacOS系统镜像文件（.ISO）。然后我直接将该ISO文件放入ventoy U盘中，在按住Option键（intel芯片）重启电脑后，可以识别EFI的磁盘，进入后也会出现ventoy的镜像选择菜单，但选择MacOS的ISO镜像后会报错“No bootfile found for UEFI! maybe the image does not support X64 UEFI”。

请问：能否通过ventoy给macbook pro重装系统？如果可以的话应该如何操作？感谢任何花时间精力回复该问题的朋友，谢谢。

**Comment by Deali-Axy** [comment]
同问，我来帮忙翻译成英文版吧。

---

I have the same question, so let me help translate it into English.

How can I use Ventoy to reinstall macOS?

Requirement: I want to reinstall macOS on my MacBook Pro 13 (2019 model) using a Ventoy USB drive.

Problem Description: The system recovery process over the internet is too slow, so I downloaded a bootable macOS system image file (.ISO). I then placed the ISO file directly onto the Ventoy USB drive. After rebooting the computer while holding down the Option key (Intel chip), the EFI disk is recognized, and the Ventoy image selection menu appears. However, when I select the macOS ISO image, an error message appears: "No bootfile found for UEFI! Maybe the image does not support X64 UEFI."

Question: Is it possible to reinstall macOS on a MacBook Pro using Ventoy? If so, how should I proceed? I appreciate any help or suggestions from those who take the time to respond. Thank you!

**Comment by wooailein** [comment]
我也想安装，目前已经下好了ISO文件，不过貌似网上都是无法直接通过ventoy安装的，不知道有没有大佬有解决方法
